tests/components/reactionsChart.logic.test.ts

```
@@ -0,0 +1,96 @@
+    const params = createSearchParams(barData, "thumbs-up");
+
+    expect(params.reactionAfter).toEqual(startOfDay(new Date("2025-07-01")));
+    expect(params.reactionBefore).toEqual(endOfDay(new Date("2025-07-01")));
+  });
+
+  it("should handle different date formats correctly", () => {
+    const testCases = [
+      "2025-01-01",
+      "2025-12-31",
+      "2025-06-15",
+    ];
```

ESLint

[error] Replace `⏎······"2025-01-01",⏎······"2025-12-31",⏎······"2025-06-15",⏎····` with `"2025-01-01",·"2025-12-31",·"2025-06-15"`

(prettier/prettier)